src/main/scala/is/hail/rvd/OrderedRVD.scala

```diff
@@ -57,9 +57,7 @@ class OrderedRVD(
       rdd.filter(p))
 
   def sample(withReplacement: Boolean, p: Double, seed: Long): OrderedRVD =
```

Comment:
I guess we're leaving this as is?  The conversation left me in what a philosophy professor called "a more refined state of confusion."

Reply:
I think we leave as is. The seed comes down from python's `sample_rows` by way of `sampleRows`. A project for someone is to stick a PRNG on python HailContext and use that instead of accepting a seed from the user. I vaguely feel like that change should also address the FunctionRegistry's randomness.